app/containers/MassEnergizeSuperAdmin/Feature Flags/AddOrEditFeatureFlags.js

```diff
@@ -0,0 +1,304 @@
+              {
+                valueToCheck: audienceKeys.ALL_EXCEPT.key,
+                fields: [
+                  {
+                    name: "community_ids",
+                    label:
+                      "Select all communities that should NOT have this feature",
+                    placeholder: "eg. Wayland",
+                    fieldType: fieldTypes.Checkbox,
+                    selectMany: true,
+                    defaultValue: comIds || [],
+                    dbName: "community_ids",
+                    data: communities,
+                  },
+                ],
+              },
+            ],
+          },
+        ],
+      },
+      // -------------------------------------
+      {
+        label: "User Audience",
+        fieldType: "Section",
+        children: [
+          {
+            name: "user_audience",
+            label: "Should this feature be available to every user?",
+            fieldType: fieldTypes.Radio,
+            isRequired: true,
+            defaultValue: userAudience || audienceKeys.EVERYONE.key,
+            dbName: "user_audience",
+            readOnly: false,
+            data: audienceKeysArr.map(([_, { name, key }]) => ({
+              id: key,
+              value: name,
+            })),
+            conditionalDisplays: [
+              {
+                valueToCheck: audienceKeys.SPECIFIC.key,
+                fields: [
+                  {
+                    name: "users",
+                    label:
+                      "Select all users that should have this feature activated",
+                    placeholder:
+                      "Search with their username, or email.. Eg. 'Mademoiselle Kaat'",
+                    fieldType: fieldTypes.AutoComplete,
+                    defaultValue: selectedUsers || [],
+                    dbName: "user_ids",
+                    data: users || [],
+                    labelExtractor: labelExt,
+                    valueExtractor: valueExt,
+                  },
+                ],
+              },
+              {
+                valueToCheck: audienceKeys.ALL_EXCEPT.key,
+                fields: [
+                  {
+                    name: "users",
+                    label: "Select all users that should NOT have this feature",
+                    placeholder:
+                      "Search with their username, or email.. Eg. 'Monsieur Brad'",
+                    fieldType: fieldTypes.AutoComplete,
+                    selectMany: true,
+                    contentType: "text",
+
+                    defaultValue: selectedUsers || [],
+                    dbName: "user_ids",
+                    labelExtractor: labelExt,
+                    valueExtractor: valueExt,
+                    data: users || [],
+                  },
+                ],
+              },
+            ],
+          },
+        ],
+      },
+      // ------------------------------------
+      {
+        label: "Feature Deactivation",
```

Comment:
Should be a separate field:
This feature does not expire   (default)
This feature expires after date

then if they choose the second radio button, they can enter the date.
Right now, a date gets added and it isn't clear how to delete it.